assets/css/style.css

```diff
@@ -107,7 +107,7 @@
 
 /*=== Border Dashed ===*/
 
-.tld-actt-border-dashed{
+.tld-actt-border-dashed, .tld-actt-dashed{
```

Comment:
@ajitbohra Instead of adding two classes, we can just modify the class in shortcode as well and keep only one. What do you think?

Reply:
keeping  one would be much better will refactor it